vendor/node-usb-native/src/detection.cpp

```diff
@@ -210,7 +210,7 @@ void StopMonitoring(const Nan::FunctionCallbackInfo<v8::Value>& args) {
 }
```

Comment:
Currently we keep two copies of the node-usb-native bits in both Arduino and Workbench extension. In future, we should consolidate this to reduce the maintain cost.

Reply:
Set up task to track this issue.https://dev.azure.com/mseng/VSIoT/_workitems/edit/1647683